### PR TITLE
Include jextract and jpackcore on all platforms

### PR DESCRIPTION
**Pull request by keithc-ca**
Fixes https://github.com/eclipse-openj9/openj9/issues/20661.

### CODE REVIEW

**Comment by pshipton** [comment]
I don't think it fixes #20661. If we are going to supply jpackcore on Windows, it needs to be in the jre/bin directory, otherwise it will be missing for anyone using the JRE. I also don't see any need to put jpackcore in the bin directory.

Given we don't create the launchers on Windows
https://github.com/ibmruntimes/openj9-openjdk-jdk8/blob/openj9/jdk/make/CompileLaunchers.gmk#L524
will this change do anything on Windows?

**Comment by keithc-ca** [comment]
This (in its current form) will not affect Windows. I see `jextract` is include in IBM Java 8, so I'll update this to do the same for Semeru.

**Comment by keithc-ca** [comment]
Updated to include the tools on Windows.

**Comment by keithc-ca** [comment]
@pshipton Any remaining concerns?

**Comment by pshipton** [comment]
Yes sorry, I made comments but never published them.

File: jdk/make/CompileLaunchers.gmk [hunk 1]
```diff
@@ -521,7 +521,7 @@ $(eval $(call SetupLauncher,traceformat, \
     -DEXPAND_CLASSPATH_WILDCARDS \
     -DJAVA_ARGS='{ "-J-ms8m"$(COMMA) "com.ibm.jvm.TraceFormat" }'))
 
-ifneq ($(findstring $(OPENJDK_TARGET_OS),aix linux macosx),)
+ifneq ($(findstring $(OPENJDK_TARGET_OS),aix linux macosx windows),)
```
[the remark this answers is not in the document]
Comment (keithc-ca) [reply jdk/make/CompileLaunchers.gmk:524]: `jextract` was initially only added for AIX and Linux (see 4b4b33d68105e39114053471a004dbccdc708033), but I can't think why it (and `jpackcore`) should not be included on all platforms.

File: closed/make/Images.gmk [hunk 2]
```diff
@@ -18,11 +18,6 @@
 # 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
 # ===========================================================================
 
-# only include jextract, jpackcore in jre/bin
```
[the remark this answers is not in the document]
Comment (keithc-ca) [reply closed/make/Images.gmk:21]: Fair enough.

**Review by pshipton — APPROVED** [review]

**Pull request merged**